sda-commons-server-circuitbreaker/src/main/java/org/sdase/commons/server/circuitbreaker/metrics/SdaCircuitBreakerMetricsCollector.java

```diff
@@ -0,0 +1,55 @@
+package org.sdase.commons.server.circuitbreaker.metrics;
+
+import io.github.resilience4j.circuitbreaker.CircuitBreaker;
+import io.github.resilience4j.circuitbreaker.CircuitBreakerRegistry;
+import io.github.resilience4j.prometheus.AbstractCircuitBreakerMetrics;
+import java.util.Collections;
+import java.util.List;
+
+/**
+ * Replaces {@link io.github.resilience4j.prometheus.collectors.CircuitBreakerMetricsCollector} but
+ * adds only resilience4j_circuitbreaker_calls_bucket metric.
+ *
+ * <p>Prometheus and Micrometer handle histogram metrics differently. Prometheus adds a
+ * resilience4j_circuitbreaker_calls_bucket histogram while micrometer adds a
+ * resilience4j_circuitbreaker_calls_count summary.
+ *
+ * <p>This class is adding only the prometheus histogram metric to not introduce a breaking changes.
+ * The rest of the metrics are added by the resilience4j micrometer library {@link
+ * io.github.resilience4j.micrometer.tagged.TaggedCircuitBreakerMetrics}.
+ *
+ * @deprecated class should be removed in the next major release. This will introduce breaking
```

Comment:
Is there a micrometer alternative or will we loose inside for circuit breakers?

Reply:
All circuit breaker metric stay the same except for `resilience4j_circuitbreaker_calls_bucket` histogram which will be replaced by  `resilience4j_circuitbreaker_calls_seconds_count` summary.

With this PR both will be available till `SdaCircuitBreakerMetricsCollector` is removed